saltwater-repl/src/commands.rs

```diff
@@ -0,0 +1,27 @@
+use crate::repl::Repl;
+use std::collections::HashMap;
+
+pub fn default_commands() -> HashMap<&'static str, fn(&mut Repl, &str)> {
+    let mut map = HashMap::<&'static str, fn(&mut Repl, &str)>::new();
+    map.insert("help", help_command);
+    map.insert("h", help_command);
+    map.insert("quit", quit_command);
+    map.insert("q", quit_command);
+    map
```

Comment:
Hmm, I was going to suggest using `map!` but it's not public and I don't know if I want to make it public ... probably not worth it for a map this small.

Reply:
This map will grow over time (I guess), so probably it is worth it

Reply:
Okay, this is unessecarry because there's now a `cmds` macro